planner/core/explain.go

```diff
@@ -89,7 +89,12 @@ func (p *PhysicalIndexScan) AccessObject() string {
 	if len(p.Index.Columns) > 0 {
 		buffer.WriteString(", index:" + p.Index.Name.O + "(")
 		for i, idxCol := range p.Index.Columns {
-			buffer.WriteString(idxCol.Name.O)
+			tableColumns := p.Table.Columns[idxCol.Offset]
```

Comment:
Please check the similar usage in about L910.

Reply:
> Please check the similar usage in about L910.

you means like `if tblColumns := p.Table.Columns[idxCol.Offset]; tblColumns.Hidden {`

Reply:
Yeah, thought LogicalIndexScan's explain info is used only for debug, it's better to modify it.

Reply:
> Yeah, thought LogicalIndexScan's explain info is used only for debug, it's better to modify it.

can you tell me how can trigger LogicalIndexScan.ExplainInfo function ?

Reply:
You don't need to test it. If you really want to check it, you need to build a LogicalIndexScan explicitly and call ExplainInfo().

Reply:
updated, please review it.

Reply:
PTAL @wjhuang2016